src/Output/RenderedContentInterface.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ * This file was originally part of the league/commonmark package.
+ *
+ * (c) Colin O'Dell <colinodell@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace UnicornFail\Emoji\Output;
+
+use UnicornFail\Emoji\Node\Block\Document;
+
+interface RenderedContentInterface extends \Stringable
```

Comment:
We needed this in commonmark as a workaround to provide additional metadata about fully-parsed-and-rendered content - specifically the front matter.  We create a new instance of an object that implements this interface but also provides a method to grab the parsed front matter.  (I'm not happy with that implementation and might be removing it completely.)

For this library, I think we can get away with the rendering step outputting a simple string.

Reply:
Perhaps. I honestly prefer passing around objects that implement interfaces rather than dealing with scalars if at all possible; especially in any sort of "extensible" environment. Given that there is `\Stringable`, it makes it easy enough to convert into a string when it ultimately needs to be. Premature rendering can often be an issue.

Reply:
This has been simplified/removed.